pyproject.toml

```diff
@@ -30,14 +30,15 @@ dependencies = [
     "setuptools",
     "tqdm",
     "numpy",
-    "ansys-dpf-gate>=0.3.*",
+    "grpcio",
+    "google-api-python-client",
 ]
 
 [project.optional-dependencies]
 plotting = ["pyvista>=0.32.0", "matplotlib>=3.2"]
 
 [tool.flit.module]
-name = "ansys.dpf.core"
+name = "ansys.dpf"
```

Comment:
@cbellot000 this tells `flit`, the build tool, that it must include everything under `ansys/dpf` when building the wheel. Otherwise it only took files under `ansys/dpf/core`.
The distinction is made with `[project] name` which declares the name of the library.